### ISSUE TITLE
Half Grate Clean Style

### ISSUE THREAD

**Issue by WhoozYerBuddy** [BEEmod/BEE2-items#2013]
I like the half-grate object.  I just wanted to mention that it is rusty in the 'Clean' style.

**Comment by Luke18033** [comment]
No it isn't...

**Comment by WhoozYerBuddy** [comment]
![20170528095605_1](https://cloud.githubusercontent.com/assets/29020282/26530567/268dd43c-438c-11e7-8979-c900a06422b0.jpg)

It sure looks rusty to me.  You think that looks 'clean'?

**Comment by Luke18033** [comment]
Those are the frames that you're supposed to use for all modern styles, but that Valve chose not to use for PeTI glass/grating for some reason. Really it's the vanilla item that should be changed, but unfortunately that's not possible in BEE.

**Comment by WhoozYerBuddy** [comment]
Ok.  Where do you come up with the idea that the rusty frames are what you're 'supposed to use in all modern styles"?  The combo doesn't look right to me.  It does to you?

Also, there's texture overlap at corners with the half grate.  It's hard to see in still images, but very obvious in-game.  That would disappear with clean frames although it would still be an issue in other styles.

![20170528102102_1](https://cloud.githubusercontent.com/assets/29020282/26530774/29a9156a-4390-11e7-84ec-fa98d8b93402.jpg)
![20170528102103_1](https://cloud.githubusercontent.com/assets/29020282/26530775/29b45d26-4390-11e7-8560-9172df93e032.jpg)
![20170528102058_1](https://cloud.githubusercontent.com/assets/29020282/26530776/29b49cc8-4390-11e7-869b-7f013ef45502.jpg)

**Comment by Luke18033** [comment]
Check the campaign, they use those frames everywhere.

**Comment by WhoozYerBuddy** [comment]
Clean workshop items are not the campaign.  The modern campaign takes place in Aperture.  The 'clean' workshop items are NEW chambers made in other locations in the  Multiverse.  Considering that you state above that you cannot change the full glass/grating frames it looks bad to have the mismatch.  You honestly think that rusty frames next to clean frames looks right??

**Comment by WhoozYerBuddy** [comment]
I thought that the 'clean' workshop items were NEW chambers made in other locations in the Multiverse, not part of the campaign in Aperture itself.  The clash of frame textures looks bad to me.  Considering you cannot change the frames for the full glass/grating panels, you honestly think that the clash looks BETTER than if the half-panel frames were clean as well??

**Comment by Luke18033** [comment]
We're not changing it...

**Comment by WhoozYerBuddy** [comment]
Okie dokie...

**Comment by awesomedoctorx3** [comment]
Yes, it _is_ very unpleasing. maybe make it an option to use a different texture, one that is not rusty?

**Comment by WhoozYerBuddy** [comment]
"Everyone knows it looks like total shit, but we're not changing it" lol!

**Comment by Luke18033** [comment]
Although the current half grate frames do look much nicer, it'd be a good idea to change them to the PeTi ones just for consistency.
